### DISCUSSION TITLE
Our First Disagreement

### DISCUSSION THREAD

**Discussion by zivy** [IBEXImagingCommunity/ibex_imaging_knowledge_base#174] [Announcements]
Unless I'm wrong and I'm never wrong - The Princess Bride.
 
One of the guiding principles of our community is **Commitment to excellence**. We strive to enable others to reproduce our work, confirming or refuting reported results. In our context, we encourage reproducing reagent validations. These contributions improve our community's shared understanding of which reagent and experimental settings will lead to successful results and more importantly, which to avoid.
 
In most cases, reproducing reagent validation leads to agreement with the existing recommendation. The more interesting cases are those where reproducing the validation study leads to a disagreement. This is how the Knowledge-Base enables our community to make science self-correcting, highlighting reagent configurations that require more careful attention.
 
A recently contributed validation reported by [0000-0002-1461-0999](https://orcid.org/0000-0002-1461-0999) for the following configuration resulted in such a disagreement:
 
| UniProt Accession Number   | Reagent Type     | Target Name / Protein Biomarker   | Target Species   | Host Organism   | Isotype   | Clonality   | Vendor    |   Catalog Number | Conjugate   | RRID       | Availability   | Method        | Tissue Preservation               | Target Tissue   | Tissue State   | Detergent         | Antigen Retrieval Conditions   | Dye Inactivation Conditions      | Recommend   | Agree                                                        | Disagree   | Contributor                                                  | Notes   |
|:---------------------------|:-----------------|:----------------------------------|:-----------------|:----------------|:----------|:------------|:----------|-----------------:|:------------|:-----------|:---------------|:--------------|:----------------------------------|:----------------|:---------------|:------------------|:-------------------------------|:---------------------------------|:------------|:-------------------------------------------------------------|:-----------|:-------------------------------------------------------------|:--------|
| P06332                     | Primary Antibody | CD4                               | Mouse            | Rat             | IgG2b     | GK1.5       | BioLegend |           100449 | BV510       | AB_2564587 | Stock          | IBEX2D Manual | 1:4 Cytofix/Cytoperm Fixed Frozen | Lymph Node      | NA             | 0.3% Triton-X-100 | NA                             | 1 mg/ml LiBH4 15 minutes + light | Yes         | [0000-0003-4379-8967](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD4_BV510/0000-0003-4379-8967.html) | [0000-0002-1461-0999](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD4_BV510/0000-0002-1461-0999.html)         | [0000-0003-4379-8967](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD4_BV510/0000-0003-4379-8967.html) |         |
 
We see that the original contributor, [0000-0003-4379-8967](https://orcid.org/0000-0003-4379-8967), recommended the use of the reagent.  Reading the [more detailed notes](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD4_BV510/0000-0003-4379-8967.html) we see that the recommendation is a bit more nuanced.
 
**Help us resolve this disagreement**:
Should this reagent validation configuration be a recommended one, possibly with certain qualifications, or should it not? Reproduce the validation study yourself and contribute your results to the Knowledge-Base.
 
p.s. Contributions to address this question should be from researchers not in the same laboratories as the existing contributors.

**Comment by KoenigJFE** [comment]
I wonder if we should have provided a note about why we felt this was not recommended. The large cells with cytosolic signal in the bottom left of the image do not have the morphology of a CD4+ T cell. We're pretty sure that is autofluorescence, and it seems to interfere with many dimmer stains on BV510 in our hands.

**Comment by radtkea** [comment]
Thanks @zivy and @KoenigJFE!!! Never been so happy to disagree or rather, come into agreement that the CD4 BV510 conjugate is suboptimal in the lymph node. 

I have encountered large, CD4+ cells in the lymph node of mice and men. These cells are enriched in the medulla. Macrophages are known to form [multinucleated giant cells](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9986822/) and CD4 is found on some human and mouse macrophages. 

**I have seen this pattern with different CD4 antibodies: Some interesting biology below.**
Human mesenteric lymph node, fixed frozen, [CD4 AF488](https://www.biolegend.com/en-us/products/alexa-fluor-488-anti-human-cd4-antibody-2727) (cyan), [CD3 AF594](https://www.biolegend.com/en-us/products/alexa-fluor-594-anti-human-cd3-antibody-9434) (red), and Hoechst (blue). This particular lymph node had several HUGE CD4+ cells that were CD3-.
![Human_FF_LN_CD3_AF594_300446_CD4_AF488_300519_Hoechst](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/bcccdb79-5a47-4325-9cb9-df90bb21691c)

Human mesenteric lymph node, FFPE, [CD4 AF488 ](https://www.abcam.com/en-us/products/primary-antibodies/alexa-fluor-488-cd4-antibody-epr6855-ab196372)(cyan), [CD3 AF555](https://www.abcam.com/en-us/products/primary-antibodies/alexa-fluor-555-cd3d-antibody-ep4426-ab208514) (red), and Hoechst (blue). Not as dramatic as the example above but reproduced using a different tissue preservation method (FFPE) and antibody clone.
![Human_FFPE_CD3_AF555_ab208514_CD4_AF488_ab196372_Hoechst](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/02ea3ec7-2e13-4047-b9dc-4016cd03d741)

Mouse inguinal lymph node, fixed frozen, [CD4 PE](https://www.bdbiosciences.com/en-us/products/reagents/flow-cytometry-reagents/research-reagents/single-color-antibodies-ruo/pe-rat-anti-mouse-cd4.553730) (cyan) and [F4/80 BV421](https://www.biolegend.com/en-us/products/brilliant-violet-421-anti-mouse-f4-80-antibody-7199) (red). This example is not as obvious as in the human lymph nodes but you do see CD4 on the medullary sinus macrophages (red).
![Mouse_LN_FF_F480_BV421_CD4_PE_553730](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/33a9aef5-8db0-4412-b856-b8ec579d5806)
